tests/python/frontend/tflite/test_forward.py

```diff
@@ -1896,6 +1896,41 @@ def test_forward_mediapipe_hand_landmark():
         tvm.testing.assert_allclose(np.squeeze(tvm_output[i]), np.squeeze(tflite_output[i]),
                                     rtol=1e-5, atol=1e-5)
 
+#######################################################################
```

Comment:
I think this should be moved so it's with the other element-wise tests and we keep the order the tests are run in the same as the order they appear in the file.

Reply:
Moved.